File: 2025-1/sesion-de-clase-18/SoftProg/SoftProgPersistencia/src/main/java/pe/edu/pucp/inf30/softprog/dao/clientes/IClienteDAO.java
package pe.edu.pucp.inf30.softprog.dao.clientes;

import pe.edu.pucp.inf30.softprog.dao.ICrud;
import pe.edu.pucp.inf30.softprog.model.clientes.Cliente;

/**
 *
 * @author eric
 */
public interface IClienteDAO extends ICrud<Cliente> {
    Cliente buscarPorDni(String dni);
}

File: 2025-2/sesion-de-clase-14/SoftProgPersistencia/DAO/Clientes/IClienteDAO.cs
using PUCP.SoftProg.Modelo.Clientes;

namespace PUCP.SoftProg.Persistencia.DAO.Clientes {
    public interface IClienteDAO : 
        IPersistible<Cliente, int> {
        Cliente BuscarPorDni(string dni);
    }
}
